kong/conf_loader/init.lua

```diff
@@ -622,6 +623,22 @@ local function infer_value(value, typ, opts)
 end
 
 
+local function try_base64_decode(vals)
```

Comment:
Perhaps try with `url` variant too, so we support them both.

Reply:
@bungle done in the last commit